include/oneapi/dpl/internal/random_impl/counter_based_engines_stuff.h

```diff
@@ -0,0 +1,99 @@
+// -*- C++ -*-
+//===-- counter_based_engines_stuff.h -------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Header file provides stuff functionality for counter-based RNG engines.
+
+#ifndef _ONEDPL_CTR_ENGINES_STUFF_H
+#define _ONEDPL_CTR_ENGINES_STUFF_H
+
+#include <concepts>
+#include <iterator>
+#include <utility>
+#include <tuple>
+#include <limits>
+
+namespace oneapi
+{
+namespace dpl
+{
+namespace internal
+{
+namespace experimental
+{
+
+/* word_mask<_U, __W> - an unsigned integral type with the low __W bits set */
+template <typename _U, unsigned __W,
```

Comment:
__W should be std::size_t instead of unsigned

Reply:
Thanks for noticing that, changed.